graylog2-web-interface/src/pages/InputDiagnosisPage.tsx

```diff
@@ -82,20 +101,22 @@ const NodeListItem = ({
 
   if (nodeId) {
     return (
-      <LinkContainer to={Routes.SYSTEM.NODES.SHOW(nodeId)}>
-        <MenuItem>
-          {nodeId && (
-            <div>
-              <b>Node ID:</b> {nodeId}
-            </div>
-          )}
-          {detailedMessage && (
-            <InputNodeInfo>
-              <b>Message:</b> {detailedMessage}
-            </InputNodeInfo>
-          )}
-        </MenuItem>
-      </LinkContainer>
+      <StyledListGroupItem>
+        <Link to={Routes.SYSTEM.NODES.SHOW(nodeId)}>
+          <>
+            {nodeId && (
+              <>
+                <b>Node ID:</b> {nodeId}
```

Comment:
```suggestion
                <strong>Node ID:</strong> {nodeId}
```

Didn't realize is before, but this should be strong instead of b to use semantic tags